fvtest/porttest/omrsockTest.cpp

```diff
@@ -417,8 +417,14 @@ TEST(PortSockTest, create_IPv6_socket_address)
 	EXPECT_EQ(OMRPORTLIB->sock_inet_pton(OMRPORTLIB, OMRSOCK_AF_INET6, "::1", addr6), 0);
 	EXPECT_EQ(OMRPORTLIB->sock_sockaddr_init6(OMRPORTLIB,  &sockAddr, OMRSOCK_AF_INET6, addr6, OMRPORTLIB->sock_htons(OMRPORTLIB, port), 0, 0), 0);
 	EXPECT_EQ(OMRPORTLIB->sock_socket(OMRPORTLIB, &socket, OMRSOCK_AF_INET6, OMRSOCK_STREAM, OMRSOCK_IPPROTO_DEFAULT), 0);
-	EXPECT_EQ(OMRPORTLIB->sock_bind(OMRPORTLIB, socket, &sockAddr), 0);
-	EXPECT_EQ(OMRPORTLIB->sock_listen(OMRPORTLIB, socket, 10), 0);
+	if (0 == (OMRPORTLIB->sock_bind(OMRPORTLIB, socket, &sockAddr))) {
+		EXPECT_EQ(OMRPORTLIB->sock_listen(OMRPORTLIB, socket, 10), 0);
+	}
+	else {
+		/* Test IPv6 support on current machine by trying to bind to ::1 localhost. Mark test as success for machines that has no IPv6 support. */
+		SUCCEED();
+		portTestEnv->log(LEVEL_ERROR, "WARNING: Cannot test IPV6: Failed to find to IPV6 interface. \n");
+	}
```

Comment:
Actually, it will need to close socket, I can change it to:
```
	if (0 != (OMRPORTLIB->sock_bind(OMRPORTLIB, socket, &sockAddr))) {
		portTestEnv->log(LEVEL_ERROR, "WARNING: Cannot test IPV6: Failed to find to IPV6 interface. \n");
                EXPECT_EQ(OMRPORTLIB->sock_close(OMRPORTLIB, &socket), 0);
                return;
	}
        EXPECT_EQ(OMRPORTLIB->sock_listen(OMRPORTLIB, socket, 10), 0);
	EXPECT_EQ(OMRPORTLIB->sock_close(OMRPORTLIB, &socket), 0);
```

Reply:
Oh sorry I thought I deleted this comment from my review--you're right and this is a bad suggestion.